include/oneapi/dnnl/dnnl_ukernel_types.h

```diff
@@ -79,7 +84,7 @@ typedef struct dnnl_transform *dnnl_transform_t;
 /// A constant transform routine handle.
 typedef const struct dnnl_transform *const_dnnl_transform_t;
 
-/// @} dnnl_api_ukernel_brgemm
+/// @} dnnl_api_ukernel_transform
```

Comment:
It was intentional as the functionality does nothing by itself without brgemm. Any specific reason to change that?

Reply:
Yes, this is to resolve a warning as per PR title/commit message :)

if this is intentional, then the other option is to get rid of dnnl_api_ukernel_transform group altogether, I suppose. For now, it is used in https://github.com/uxlfoundation/oneDNN/blob/main/include/oneapi/dnnl/dnnl_ukernel.hpp#L400 for instance, so this commit also aligns to that usage.

Reply:
Alright, the copy-paste approach failed at some point. Let's keep the new version and see how it goes. Thanks for checking it out.